mmpose/datasets/pipelines/hand_transform.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+
+from mmpose.datasets.registry import PIPELINES
+from .top_down_transform import TopDownRandomFlip
+
+
+@PIPELINES.register_module()
+class HandRandomFlip(TopDownRandomFlip):
+    """Data augmentation with random image flip.
+
+    Required keys: 'img', 'joints_3d', 'joints_3d_visible', 'center',
+    'hand_type', 'rel_root_depth' and 'ann_info'.
+    Modifies key: 'img', 'joints_3d', 'joints_3d_visible', 'center',
+    'hand_type', 'rel_root_depth'.
+
+    Args:
+        flip_prob (float): Probability of flip.
+    """
+
+    def __init__(self, flip_prob=0.5):
+        super().__init__(flip_prob=flip_prob)
+
+    def __call__(self, results):
+        """Perform data augmentation with random image flip."""
+        # base flip augment
+        super().__call__(results)
+
+        # flip hand type and root depth
+        hand_type = results['hand_type']
+        rel_root_depth = results['rel_root_depth']
+        if self.flip_flag:
+            hand_type[0], hand_type[1] = hand_type[1], hand_type[0]
+            rel_root_depth = -rel_root_depth
+        results['hand_type'] = hand_type
+        results['rel_root_depth'] = rel_root_depth
+        return results
+
+
+@PIPELINES.register_module()
+class HandGenerateRelDepthTarget:
+    """Generate the target relative root depth.
+
+    Required keys: 'rel_root_depth', 'rel_root_valid', 'ann_info'. Modified
+    keys: 'target', 'target_weight'.
+    """
+
+    def __init__(self):
+        pass
+
+    def __call__(self, results):
+        """Generate the target heatmap."""
+        rel_root_depth = results['rel_root_depth']
+        rel_root_valid = results['rel_root_valid']
+        cfg = results['ann_info']
+        D = cfg['heatmap_size_root']
+        depth_size = cfg['bbox_depth_size_root']
```

Comment:
What does `depth_size` mean?

Reply:
It means the depth size of heatmap in camera space.
For example, setting 'bbox_depth_size_root' to 400.0 means that the 1D heatmap corresponds to the depth range [-200, +200].

Reply:
Yeah the name "size" is too general. How about replacing it with "bound"? Does "bbox_depth_bound" sounds better? @zengwang430521 @jin-s13

Reply:
OK, I will change it to 'heatmap_depth_bound'.